common/src/types.rs

```diff
@@ -97,13 +98,17 @@ pub enum Diff {
     General(Box<ReservedState>, Hash256),
 }
 
+/// A general transaction to be included in the agenda.
+///
+/// Note that `author` and `timestamp` of a general transaction is not cared by the Simperby protocol;
```

Comment:
How about specifying what they are handled by? Git?

I do not think just specifying that they are not handled by Simperby is not enough.

Reply:
Elaborated the description